test/name_service/api.clj

```diff
@@ -32,4 +35,72 @@
          "X-XSS-Protection" "1; mode=block"
          "X-Download-Options" "noopen"
          "X-Permitted-Cross-Domain-Policies" "none"
-         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
\ No newline at end of file
+         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
+
+(deftest test-delete-keymap-wrong-params
+  (is (= 400 (:status (response-for service
+                                    :delete "/"
+                                    :headers headers
+                                    :body "{\"where\":{ \"jira\" : 2}}")))))
```

Comment:
Maybe it could be better to create some kind of helper which would take map as parameter and then encode to json with cheshire instead of encoding JSON by yourself. What do you think?